src/controllers/user.js

```diff
@@ -173,5 +173,25 @@ export default {
       page,
       message: 'Request successful',
     });
+  },
+
+  /**
+   * controller for getting all bookmarks for a user
+   *
+   * @function
+   *
+   * @param {Object} request - express request object
+   * @param {Object} response - express response object
+   *
+   * @returns {Object} - all bookmark for a user
+   */
+  getAllBookmark: async (request, response) => {
+    const { id: userId, limit = 10, page = 1 } = request.user;
+
+    const data = await paginator(Bookmarks, { where: { userId }, page, limit });
+
+    return response
```

Comment:
Where does the error in your screenshot come in?

Reply:
From the paginator function.

Reply:
I think this has been fixed. Please confirm.